pkg/operation/botanist/component/kubeapiserver/networkpolicies.go

```diff
@@ -178,7 +183,7 @@ func (k *kubeAPIServer) reconcileNetworkPolicyAllowKubeAPIServer(ctx context.Con
 				}},
 				Ports: []networkingv1.NetworkPolicyPort{{
 					Protocol: &protocol,
-					Port:     &portVPNSeedServer,
+					Port:     port,
```

Comment:
I was wondering, why it was working with the wrong port. Then I found the networkpolicy `allow-to-private-networks` that allows egress traffic to all private ip addresses. Shouldn't we remove this egress rule here, as it has no effect?

Reply:
The networkpolicy `allow-to-private-networks` allows egress traffic to all private IP addresses, but not to the seed networks. So this egress rule is essential at least for the local provider.